sui_types/src/waypoint.rs

```diff
@@ -0,0 +1,445 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
```

Comment:
Why aren't we using a standard accumulator from some library? Is there some special functionality we need for checkpoints? 

If so, should we define an API so that we can switch accumulator implementations in the future?

Reply:
Is there such a library?

Reply:
This is not a generic accumulator. It is a muhash that allows for set operations, but not individual membership / non-membership queries.